Devops tools

romani opened the pull request:
introduction of devops tools

rnveach commented:
@daniilyar Since we now have a `hyper.sh`, we wanted to start playing with Jenkins and possibly looking into adding it as one of our CIs. @romani stated you have credentials and a script already for checkstyle a Jenkins. If so, can you please share everything with us?

daniilyar commented:
Hi @rnveach! All stuff is in DevOps branch of contribution repo. All credentials are Synced with @romani and are present on Checkstyle internal email account. Jenkins is accessible via Github auth to all GitHub organization members. Let's have a sync call so I can show you everything that I have  so far.

romani commented:
> All stuff is in devops branch of contribution repo.

branch was merged and deleted, I changed jenkins jobs that pointed to this branch to point to master.
